docs/attributes-registry/android.md

```diff
@@ -13,16 +13,16 @@
 
 The Android platform on which the Android application is running.
 
-| Attribute | Type | Description | Examples | Stability |
-|---|---|---|---|---|
+| Attribute                                                                            | Type   | Description                                                                                                                                                                                                                               | Examples   | Stability                                                        |
```

Comment:
these files are auto-generated, we should not modify them (and I think whitespaces in jinja are too complicated to format them in this way)

Reply:
@lmolkova `make fix-format` will update the tables